### PR TITLE
Slack documentation

### PR DESCRIPTION
**Pull request by KarolJaksik**
Our slack connector had no documentation so it had to be created. It includes a README and an installation guide.

### CODE REVIEW

File: chart/slackconnector/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Slack Connector <!-- omit in toc -->
+
+## Overview
+
+The chart creates a Slack Connector Deployment and a Namespace in Kyma.
+Moreover it creates an application and a service which is binded to the newly created namespace.
```
Comment (majakurcius) [inline chart/slackconnector/README.md]:
```suggestion
Moreover, it creates an application and a service which is bound to the newly created Namespace.
```

Regarding `is`: is it just the service that IS bound, or is it both the service AND the application that ARE bound?

File: docs/slack-connector/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Slack Connector Installation<!-- omit in toc -->
+
+- [Overview](#overview)
+- [Installation in Kyma with Helm](#installation-in-kyma-with-helm)
+  - [Prerequisites](#prerequisites)
+  - [Steps](#steps)
+
+## Overview
+
+Slack Connector is a component which allows contact from inside of Kyma environment to the Slack API.
```
Comment (majakurcius) [inline docs/slack-connector/README.md]:
```suggestion
Slack Connector is a component which allows contact from the inside of the Kyma environment to the Slack API.
```
How about:
> ... which allows the Kyma environment to communicate with the Slack API. 

or 
> ... which allows for communication between the Kyma environment and the Slack API. 

?

File: docs/slack-connector/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## Installation in Kyma with Helm
+
+### Prerequisites
+
+- Connection to Kyma cluster
+- Slack Connector Docker image
+
+### Steps
+
+1. Visit our [authentication page](https://auth-slack.herokuapp.com/). Click `Add to Slack`. You will be redirected to another page. Select your desired workspace and click `Allow`
```
Comment (majakurcius) [inline docs/slack-connector/README.md]:
```suggestion
1. Go to the [authentication page](https://auth-slack.herokuapp.com/). Click `Add to Slack`. This redirects you to another page. Select your desired workspace and click `Allow`.
```
- Try to avoid using possessive pronouns like "our", "my", etc.
- Don't use future tense (_will_), use present tense.
- Don't use passive voice (_you are redirected_), use active voice (_this redirects you_).
- Punctuate your sentences. Remember to add periods at the end of your sentences. If you use sentences in a list, end them with periods.

File: docs/slack-connector/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    >**NOTE:** If the link doesn't work it means that you have to create an application like that yourself. The tutorial for that can be found in the Slack API [documentation](https://api.slack.com/docs/oauth#flow).
```
Comment (majakurcius) [inline docs/slack-connector/README.md]:
```suggestion
    >**NOTE:** If the link does not work, it means that you have to create an application like that yourself. To create an application, see [this tutorial](https://api.slack.com/docs/oauth#flow) in the Slack API documentation.
```
- Don't use contractions, use full words (_do not_ instead of _don't_). 
- I'm not sure what you mean by 
> ... an application like that

I think you need to be more specific. For example:
> If the link does not work, it means that an application that [does something - specify what it is it does] does not exist and you have to create it yourself. 

or 

>  If the link does not work, it means that there is no application that [does something - specify what it is it does].

and then:
> To create an application [that does X], see ...

File: docs/slack-connector/README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+2. Copy the authentication token. You will need it later in the Helm command.
+3. Go to [Kyma repository](https://github.com/kyma-project/kyma) and run script `/installation/scripts/tiller-tls.sh` to get certificates needed for using Helm commands. By default they are stored in `~/.helm` directory. After that add `--tls` flag to every Helm command to authorize and authenticate yourself
```
Comment (majakurcius) [inline docs/slack-connector/README.md]:
```suggestion
3. Go to [Kyma repository](https://github.com/kyma-project/kyma) and run the script `/installation/scripts/tiller-tls.sh` to get certificates needed for using Helm commands. By default they are stored in the `~/.helm` directory. After that, add `--tls` flag to every Helm command to authorize and authenticate yourself.
```
I'd consider changing `yourself` to `user` or something similar. Is it the user that you authenticate or an app or what? I think you need to be more specific and "yourself" is too colloquial.

File: docs/slack-connector/README.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+4. Go to `chart/slackconnector` directory. Run the command to install Slack Connector:
```
Comment (majakurcius) [inline docs/slack-connector/README.md]:
```suggestion
4. Go to the `chart/slackconnector` directory. Run this command to install Slack Connector:
```

File: docs/slack-connector/README.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+    ``` shell
+    helm install --set container.image={DOCKER_IMAGE} --set kymaAddress={KYMA_ADDRESS} --set slackBotToken={SLACK_TOKEN} -n {RELEASE_NAME} . --tls
+    ```
+
+    >**CAUTION:** Kyma address should be in the right format. It must consist of domain name, without dot  character at the beggining, for example `35.187.32.214.xip.io`
```
Comment (majakurcius) [inline docs/slack-connector/README.md]:
```suggestion
    >**CAUTION:** Make sure that the Kyma address is in the correct format. It must consist of the domain name, without the dot character at the beginning. For example, `35.187.32.214.xip.io`.
```
`It must` or `it consists`?
- Don't use `should`.
- Don't use `for example` in the middle of the sentence.
- I'd rephrase that:
> It must consist of the domain name, **without the dot character at the beginning**.

to something like:
>  It consists of the domain name and omits the dot at the beginning. 

if the dot is there in other cases and just here needs to be omitted.

File: slack-connector/README.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
+# Slack Connector
+
+## Overview
+
+Slack Connector is a component which allows contact from inside of Kyma environment to the Slack API.\
```
Comment (majakurcius) [inline slack-connector/README.md]:
```suggestion
Slack Connector is a component which allows contact from inside of Kyma environment to the Slack API.
```
See the second comment. Copy that from the other file.

File: slack-connector/README.md [hunk 9]
```diff
@@ -0,0 +1,6 @@
...
+It registers an application in the Application Registry. The installation guide can be found [here](/docs/slack-connector/README.md)
```
Comment (majakurcius) [inline slack-connector/README.md]:
```suggestion
It registers an application in the Application Registry. The installation guide can be found [here](/docs/slack-connector/README.md)
```
The guidelines say you shouldn't use passive voice (although I do agree that `the installation guide can be found here` sounds good).
Maybe rephrase that to something like:
> See [the installation guide](/docs/slack-connector/README.md).

or 

> See the installation guide [here](/docs/slack-connector/README.md).

?

File: chart/slackconnector/README.md (hunk 1 above)
Comment (KarolJaksik) [reply chart/slackconnector/README.md]: Only the service, you have to bind the application yourself
